.ci/gcb-generate-diffs.yml

```diff
@@ -182,6 +182,45 @@ steps:
           - $PROJECT_ID
           - "20"  # Build step
 
+    - name: 'gcr.io/graphite-docker-images/terraform-validator-tester'
+      id: tfv-test
```

Comment:
It looks like this section duplicates the previous one

Reply:
resolved